src/utils/queryFilters.ts

```diff
@@ -0,0 +1,51 @@
+import qs from 'qs'
+import { isArray, mapKeys, isNil, isObject, isBoolean, isEmpty, omitBy } from 'lodash'
```

Comment:
You don't like the native features? 🥺 like Array.isArray(arr), Object.keys(obj), typeof...